src/js/plugins/plugin.mobile_nav.js

```diff
@@ -9,122 +9,122 @@
-            var $drawerEl = this.buildMobileDrawerElement();
-            this.refs.$br.append($drawerEl);
-
-            // Render info into mobile info before mmenu
-            this.buildInfoDiv(this.$('.BRmobileInfo'));
-            this.buildShareDiv(this.$('.BRmobileShare'));
-
-            var $mmenuEl = $drawerEl;
-            $mmenuEl.mmenu({
-              navbars: [
-                 { "position": "top" },
-              ],
-              navbar: {
-                add: true,
-                title: this.mobileNavTitle,
-                titleLink: 'panel'
-              },
-              extensions: [ "panelshadow" ],
-            }, {
-              offCanvas: {
-                wrapPageIfNeeded: false,
-                zposition: 'next',
-                pageSelector: this.el,
-              }
-            });
-
-            var $BRpageviewField = $mmenuEl.find('.BRpageviewValue');
-            $mmenuEl.data('mmenu').bind('opened', function() {
-                // Update "Link to this page view" link
-                if ($BRpageviewField.length)
-                    $BRpageviewField.val(window.location.href);
-            });
-
-            // High contrast button
-            $drawerEl.find('.high-contrast-button').click(function() {
-                self.refs.$br.toggleClass('high-contrast');
-            });
-
-            // Bind mobile switch buttons
-            $drawerEl.find('.DrawerLayoutButton.one_page_mode').click(function() {
-                self.switchMode(self.constMode1up);
-            });
-            $drawerEl.find('.DrawerLayoutButton.two_page_mode').click(function() {
-                self.switchMode(self.constMode2up);
-            });
-            $drawerEl.find('.DrawerLayoutButton.thumbnail_mode').click(function() {
-                self.switchMode(self.constModeThumb);
-            });
-
-            if (this.mobileNavFullscreenOnly) {
-                $(document.body).addClass('BRbodyMobileNavEnabledFullscreen');
-            } else {
-                $(document.body).addClass('BRbodyMobileNavEnabled');
-            }
-
-            this.refs.$mmenu = $mmenuEl;
+  return function (mode, ui) {
+    let self = this;
+
+    if (this.enableMobileNav) {
+      let $drawerEl = this.buildMobileDrawerElement();
+      this.refs.$br.append($drawerEl);
+
+      // Render info into mobile info before mmenu
+      this.buildInfoDiv(this.$('.BRmobileInfo'));
+      this.buildShareDiv(this.$('.BRmobileShare'));
+
+      var $mmenuEl = $drawerEl;
+      $mmenuEl.mmenu({
+        navbars: [
+          { "position": "top" },
+        ],
+        navbar: {
+          add: true,
+          title: this.mobileNavTitle,
+          titleLink: 'panel'
+        },
+        extensions: [ "panelshadow" ],
+      }, {
+        offCanvas: {
+          wrapPageIfNeeded: false,
+          zposition: 'next',
+          pageSelector: this.el,
         }
+      });
+
+      let $BRpageviewField = $mmenuEl.find('.BRpageviewValue');
```

Comment:
Oh drats! Looks like `estlint` replaced all the `var`s with `let` -\_-`; some of these can/should be `const`; I'll see if I can add the `prefer-const` rule to the eslint config.

Reply:
I fixed them :)